### ISSUE TITLE
paper-progress-circular doesn't rotate anymore

### ISSUE THREAD

**Issue by pomm0** [miguelcobain/ember-paper#173]
Hey,

i've seen that the paper-progress-circular is not rotating anymore. Same on your demosite http://miguelcobain.github.io/ember-paper/#/progress-circular.

It's working on the Angular-Material demosite https://material.angularjs.org/latest/#/demo/material.components.progressCircular

Not sure what happend that it's not working anymore.

Testet on Chrome Linux Version 45.0.2454.85 (64-bit) and IE 11 Win 7

**Comment by cah-danmonroe** [comment]
Demo is working for me on my MacBook - Chrome Version 45.0.2454.85 (64-bit)

**Comment by miguelcobain** [comment]
Working here too!

**Comment by pomm0** [comment]
I can't test it on OSX. But i tested it on a iPhone and in Win 7 with Version 45.0.2454.85 m and it does not work. It does the Animation of shrinking and growing but it doesn't rotate.

**Comment by pomm0** [comment]
On Win 7 Chrome Version 45.0.2454.85 m and everywhere else i tried its doing like that:

https://vid.me/VfIr

**Comment by miguelcobain** [comment]
Ah, rotating!

Yes, I can see the issue as well!

**Comment by hoboman313** [comment]
The working demo has been moved to: https://material.angularjs.org/latest/demo/progressCircular

This has been a big pet peeve for me. Has anyone looked into how it broke?

**Comment by mike1o1** [comment]
I'll look this week and should be able to get it fixed.

**Comment by hoboman313** [comment]
Hopefully this saves you some time, but the circleStyle computed property seems to be the culprit responsible for this breaking inline style.
I tried to patch around it using:
md-progress-circular[md-mode=indeterminate] .md-spinner-wrapper {
  transform: none !important;
}
and this does make the spinner spin correctly for some browsers.
Tested and works for - IE 11, Chrome 45. Tested and broken in FF 41. Basically the 'transform' property of .md-spinner-wrapper needs to be computed to 'matrix(...)' for the spinning to work correctly.
